gandi/cli/tests/commands/test_contact.py

```diff
@@ -152,5 +152,7 @@ def test_create_apikey_ok(self):
 Will save your apikey into the config file.""")
         self.assertEqual(result.exit_code, 0)
 
-        self.assertEqual(GandiModule._conffiles,
```

Comment:
Though I find this test a little annoying the way it is right now, because it always fails if you have a local configuration file. I think the way I had changed it avoided that while keeping the test's objective.